classes/class-wc-connect-nux.php

```diff
@@ -400,14 +400,16 @@ public function get_jetpack_redirect_url() {
 		}
 
 		public function set_up_nux_notices() {
-			if ( ! current_user_can( 'manage_woocommerce' )
-				|| ! current_user_can( 'install_plugins' )
-				|| ! current_user_can( 'activate_plugins' )
-			) {
```

Comment:
Let's add a comment here or below explaining why we're checking what we're checking and how it relates to both single site and multi site installations.